server/views/pages/recallEntry/enter-recall-type.njk

```diff
@@ -0,0 +1,42 @@
+{% extends "../../partials/prisonerBannerLayout.njk" %}
+
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% block content %}
+
+  {{ govukRadios({
+    name: "recallType",
+    fieldset: {
+      legend: {
+        text: "What is the recall type?",
+        isPageHeading: true,
+        classes: "govuk-fieldset__legend--l"
+      }
+    },
+    items: [
```

Comment:
These recall-types are hard-coded here for the POC, this isnt yet an exhaustive list and there is potential to derive this based on the sentences. So WIP